src/test/java/BasePage.java

```diff
@@ -0,0 +1,12 @@
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class BasePage {
+    WebDriver driver;
+
```

Comment:
```suggestion
@BeforeTest
```

Add annotation please

Reply:
Changed it